vizro-core/docs/pages/user-guides/card-button.md

```diff
@@ -590,6 +590,27 @@ If you use the image URL hash `icon-top`, the image will be styled according to
 Note that in the above example the first [`Card`][vizro.models.Card] navigates to an existing [`Page`][vizro.models.Page]
 in the app with `path = filters-and-parameters` and the second one to an external link.
 
+#### Icon responsive to theme switch
+
+To add an icon that is responsive to the theme switch, you will need to override the value of the `filter` property.
+
+The `filter` property allows the application of visual effects to elements and accepts various functions as parameters. In the example below, we will use `invert()` function, to invert the color of the icon on theme switch.
+
+1. Begin by introducing a new variable into the `vizro_dark` theme.
+```css
+.vizro_dark {
+  --icon-color: invert(100%);
+}
```

Comment:
I don't think this would actually work if they only add this change to a custom CSS file. I think they would have to copy all the other values as well. Can you double-check? And add a sentence here that they need to add the CSS to their assets folder and then link to our docs on how to add CSS into the assets folder.

I think this is actually something that users might use commonly, so I would even suggest to just add that variable to our source code. Maybe call it `--inverse-color` ? And then update the instructions because then they don't need to add any custom CSS anymore.